oozie-to-airflow/README.md

```diff
@@ -272,33 +294,25 @@ edit connections so you must add one like:
 
 `airflow connections --add --conn_id <SSH_CONN_ID> --conn_type SSH --conn_password <PASSWORD>`
 
-More information can be found on [Airflow's Website](https://airflow.apache.org/cli.html#connections)
+More information can be found on [Airflow's Website](https://airflow.apache.org/cli.html#connections).
```

Comment:
```suggestion
More information can be found on [Airflow's Documentation](https://airflow.apache.org/cli.html#connections).
```
The Airflow website does not exist yet. It's WIP.

Reply:
This was probably the previous author's "shortcut" for referring to the documentation. But I changed it to be absolutely clear.